tavla/next.config.js

```diff
@@ -19,8 +19,12 @@ const nextConfig = {
                 hostname: 'firebasestorage.googleapis.com',
             },
         ],
+        dangerouslyAllowSVG: true,
     },
     experimental: {
+        serverActions: {
+            bodySizeLimit: '10mb',
+        },
```

Comment:
Jeg tror ikke vi trenger å sette den så høy. Man får til å laste opp logoer nå i prod som hvertfall er 4.8 MB (jeg testa med et PNG-bilde). Den er default satt til 1 MB. Vi må også være litt forsiktig med å sette denne, er nok en grunn til at den er default til 1MB av sikkerhetsårsaker: https://nextjs.org/docs/app/api-reference/next-config-js/serverActions#bodysizelimit

Tenkte på om det var mulig å tillate kun at en spesifikk server-action fikk ha større bodySizeLimit? Altså den server actionen som laster opp filen.

Reply:
ja, så den lenken. Man kan eventuelt lage en api route som håndterer file upload i stedet, med spesifikk bodysizelimit 🤔

Reply:
Det kunne jo vært en løsning, også heller skru av SVG for nå og lage en oppgave på at filopplastning må skje på en spesifikk rute

Reply:
Vet vi da hva som skjer med svg-er som allerede ligger i databasen? Er det bare en sjekk på frontend før man sender inn som stopper folk fra å sende inn svg-er? Tror dette har vært et problem en stund, denne loggen er fra slutten av oktober og den "foreslår" jo dangerouslyallowsvg 🤔 
![Screenshot 2024-11-13 at 07 59 24](https://github.com/user-attachments/assets/f8c58020-408f-4847-8861-0d40edb5584c)

Reply:
skal være fjernet nå, siden det går gjennom API route. Har også på dangerouslyallowxml, men sanitizer disse filene i tillegg til å sette Content Security Policies https://nextjs.org/docs/pages/api-reference/components/image#dangerouslyallowsvg